README.md

```diff
@@ -93,6 +93,13 @@ $ ./x.py build # `./x.py build -h` to check more options;
                # especially, `./x.py build --ghproxy` will fetch dependencies via ghproxy.com.
 ```
 
+To build with TLS support, you'll need OpenSSL development libraries (e.g. libssl-dev on Debian/Ubuntu) and run:
```

Comment:
Should we remove openssl packages like `libssl-dev` in Prerequisite above?

Reply:
Does it sound good to preserve the OpenSSL in prerequisite? I think it's ok since it's a common library and harms nothing.